src/js/tracks/text-track-display.js

```diff
@@ -319,6 +319,33 @@ class TextTrackDisplay extends Component {
       }
       this.updateForTrack(descriptionsTrack);
     }
+
+    if (!window.CSS.supports('inset', '10px')) {
+      const textTrackDisplay = this.el_;
+      const vjsTextTrackCues = textTrackDisplay.querySelectorAll('.vjs-text-track-cue');
+
+      // Clear inline style before getting actual height of textTrackDisplay
+      textTrackDisplay.style = '';
+      const textTrackDisplayHeight = textTrackDisplay.getBoundingClientRect().height;
+
+      // vjsTextTrackCue style updates
+      if (vjsTextTrackCues.length > 0) {
+        vjsTextTrackCues.forEach((vjsTextTrackCue) => {
+          const insetStyles = vjsTextTrackCue.style.inset.split(' ');
```

Comment:
Is it possible for the `inset` style to not be set on the cue? If so, we might want a check to isure it exists before we move forward.

Reply:
In every case I have tried those are always there, nevertheless is a good thing what you mentioned I updated the PR to ensure those exists before we use those to avoid any error